src/profile.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: Profile
+nav_order: 6
+---
+
+# Profile
+{:.no_toc}
+
+A profile represents a centrapay user's attributes in the cognito user pool.
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+
+## Updating a user's profile
+
+{% endpoint POST https://service.centrapay.com/api/merchants %}
+
+```sh
+curl -X POST "https://service.centrapay.com/api/me/profile/update" \
+  -H "authorization: $jwt" \
+  -H "content-type: application/json" \
+  -d '{
+    "givenName": "John",
+    "familyName": "Doe",
+    "featureUpdates": "false",
+    "marketingUpdates": "true",
+    "email": "john.doe@centrapay.com"
+  }'
+```
+
+**Optional Fields** (Note at least one field must be provided in the request)
+
+|      Field       |  Type  |                 Description                                                                        |
+| :--------------- | :----- | :------------------------------------------------------------------------------------------------- |
+| givenName        | String | First name                                                                                         |
+| familyName       | String | Last name                                                                                          |
+| featureUpdates   | String | Whether a user would like to receive emails for feature updates (valid values are true or false)   |
+| marketingUpdates | String | Whether a user would like to receive emails for marketing updates (valid values are true or false) |
```

Comment:
these should be boolean

Reply:
Updated the docs to be boolean need to update the API, the decision for them to be strings was based on them being persisted as a string in cognito but boolean makes more sense.